tests/tokenWrapper.test.ts

```diff
@@ -0,0 +1,800 @@
+
+
+        let initialSenderWrappedBalance = await tokenWrapperContract.query.psp22Balance(sender.address);
+        let initialContractBalance = await tokenWrapperContract.query.psp22ContractBalance();
+        let initialFeeRecipientBalance = await tokenWrapperContract.query.psp22Balance(newFeeRecipient);
+
+        expect(Number(initialSenderWrappedBalance.output)).to.not.equal(0);
+        expect(Number(initialContractBalance.output)).to.equal(0);
+        expect(Number(initialFeeRecipientBalance.output)).to.equal(0);
+
+        // now do wrapping
+        let wrapFunction = await tokenWrapperContract.tx.wrap( BobSigner.address, 10, { value: 10 });
+        expect(wrapFunction).to.be.ok;
+
+        // to validate that psp22 token has been minted for sender
+        let senderWrappedBalanceAfter = await tokenWrapperContract.query.psp22Balance(sender.address);
+        // to validate that psp22 token funds was transferred to the contract
+        let contractBalanceAfter = await tokenWrapperContract.query.psp22ContractBalance();
+        // to validate that fee has been transferred to the fee recipient
+        let feeRecipientBalanceAfter = await tokenWrapperContract.query.psp22Balance(feeRecipient);
+
+
+        expect(Number(senderWrappedBalanceAfter.output)).to.not.equal(0);
+        expect(Number(contractBalanceAfter.output)).to.not.equal(0);
+        expect(Number(feeRecipientBalanceAfter.output)).to.not.equal(0);
+
+        //expect(Number(senderWrappedBalanceAfter.output)).to.not.equal(Number(initialSenderWrappedBalance.output));
+        expect(Number(contractBalanceAfter.output)).to.be.greaterThan(Number(initialContractBalance.output));
+        expect(Number(feeRecipientBalanceAfter.output)).to.be.greaterThan(Number(initialFeeRecipientBalance.output));
+
+        //expect(await tokenWrapperContract.tx.terminateContract(DaveSigner.address)).to.be.ok
+
+    });
+
+    it('Test native wrapping for functionality', async () => {
+
+        let initialSenderWrappedBalance = await tokenWrapperContract.query.psp22Balance(FerdieSigner.address);
+        let initialContractBalance = await tokenWrapperContract.query.nativeContractBalance();
+
+
+        expect(Number(initialSenderWrappedBalance.output)).to.equal(0);
+        expect(Number(initialContractBalance.output)).to.equal(0);
+
+        let wrapFunction = await tokenWrapperContract.tx.wrapFor( null, FerdieSigner.address, 0, { value: 1000 });
+
+        expect(wrapFunction).to.be.ok;
+
+        // to validate that psp22 token has been minted for sender, in this case Ferdie
+        let senderWrappedBalanceAfter = await tokenWrapperContract.query.psp22Balance(FerdieSigner.address);
+        // to validate that native funds was transferred to the contract
+        let contractBalanceAfter = await tokenWrapperContract.query.nativeContractBalance();
+
+        expect(Number(senderWrappedBalanceAfter.output)).to.not.equal(0);
```

Comment:
Can we avoid these types of checks? Let's use strict equality if possible. Using variables should make this possible since in this test you wrap 1000 tokens.